modules/spark/src/test/scala/org/apache/ignite/spark/IgniteSQLDataFrameSpec.scala

```diff
@@ -284,5 +294,27 @@ class IgniteSQLDataFrameSpec extends AbstractDataFrameSpec {
             .load()
 
         personDataFrame.createOrReplaceTempView("person")
+
+        createEmployeeCache(client, "employeeCache1")
+
+        spark.read
```

Comment:
Le'ts move creation of this data frame inside the test.
It will simplify understanding.